src/mt/mod.rs

```diff
@@ -0,0 +1,166 @@
+//! Mobile Terminated
+//!
+//!
+
+/*
+Information Element Identifiers:
+    0x01 MO Header IEI
+    0x02 MO Payload IEI
+    0x03 MO Lat/Lon Location Information IEI
+    0x05 MO Confirmation IEI
+
+    0x41 MT Header IEI
+    0x42 MT Payload IEI
+    0x43 MT Lat/Lon Location Information IEI
+    0x44 MT Confirmation Message IEI
+    0x45 MT LAC/Cell ID Location Informatio IEI
+
+
+Protocol Revision Number        1   1
+Overall Message Length          2   97
+MT Header IEI                   1   0x41
+MT Header Length                2   21
+Unique Client Message ID        4   “Msg1”
+IMEI (User ID)                  15  300034010123450
+MT Disposition Flags            2   0x0000
+MT Payload IEI                  1   0x42
+MT Payload Length               2   70
+MT Payload                      70  Payload Bytes
+*/
```

Comment:
Can these be put into the module-level documentation and formatted as a markdown table? Thanks.

Reply:
Done.